src/calliope/backend/latex_backend_model.py

```diff
@@ -421,14 +434,17 @@ def add_parameter(  # noqa: D102, override
             "title": self._PARAM_TITLES.get(parameter_name, None),
             "description": self._PARAM_DESCRIPTIONS.get(parameter_name, None),
             "unit": self._PARAM_UNITS.get(parameter_name, None),
+            "type": self._PARAM_TYPE.get(parameter_name, None),
```

Comment:
No, because as it currently stands you can define any new parameter you like. The lack of associated schema just means that it won't have things like a type or a default value (which means you have to be careful with the math that parameter is used in). Agreed that this will be easier when we have user-define parameters, but for now this makes sense.